### PR TITLE
[Workflow] Explain how to style a workflow dump

### LINKED ISSUE #11675: Add docs for: [Workflow] Add colors to workflow dumps
Feature PR: https://github.com/symfony/symfony/pull/29538

Now that #9476 has been fixed by superseded by #11209, and this last one is merged, we can document how to add colors.

For reference, see this nice example from @Korbeil :smiley: : https://twitter.com/Korbeil_/status/1135914174603714565

### PR DESCRIPTION
**Pull request by alexisLefebvre**
Fixes #11675

Explain what was added in https://github.com/symfony/symfony/pull/29538

TODO:

- [x] Explain keys
- [x] Add YAML format
- [x] Add XML format
- [x] Add PHP format

### CODE REVIEW

File: workflow/dumping-workflows.rst [hunk 1]
```diff
@@ -54,5 +54,247 @@ files and ``PlantUmlDumper`` to create the PlantUML files::
     $ php dump-graph.php | dot -Tsvg -o graph.svg
     $ php dump-graph.php | java -jar plantuml.jar -p  > graph.png
 
+Styling
+-------
+
+You can use `metadata` with the following keys to style the workflow:
```
Comment (OskarStark) [inline workflow/dumping-workflows.rst]: Pleas double backticks around metadata

File: workflow/dumping-workflows.rst [hunk 2]
```diff
@@ -54,5 +54,247 @@ files and ``PlantUmlDumper`` to create the PlantUML files::
...
+
+* for places:
+  * `bg_color`: a color
```
Comment (OskarStark) [inline workflow/dumping-workflows.rst]: Same for the others

File: workflow/dumping-workflows.rst (hunk 1 above)
Comment (javiereguiluz) [reply workflow/dumping-workflows.rst]: Done while merging. Thanks.

**Comment by javiereguiluz** [comment]
Thank you Alexis! In 7a084325387e385ff3b6b7244441937fabe1ff88 we made some syntax fixes and some minor changes.

**Comment by alexislefebvre** [comment]
Nice! Thanks.

**Pull request merged**